atlp-rwanda/eagles-phantom-frontend
# #6yTQ5Oqv Landing Page

### What does this PR do?
-building landing page or homepage of phantom project

### description of tasks to be completed
-implementing landing page
-landing page is responsive
-page has important links and app info
-Skelton screen when app is loading

### How can be manually tested?
-clone this repo
-run `npm install `
-run `npm run dev`
-see the opened link in your default browser

### Relavant trello story
[](https://trello.com/c/6yTQ5Oqv/28-landing-page)
